src/main/resources/static/js/main.js

```diff
@@ -150,47 +149,65 @@
         latitude: position.coords.latitude,
         longitude: position.coords.longitude
       });
+      geometry=true;
       locationButton.toggleClass('loading');
     }, function () {
       locationButton.toggleClass('loading');
       locationButton.toggleClass('disabled');
     })
   });
 
-  const geocodingButton = $('.ui.button#geocoding'); // it is not used
   const card = $('#pac-card');
   const input = document.getElementById('pac-input');
-  //const input = $('.ui.input#pac-input');
   const infowindowContent = document.getElementById('infowindow-content');
-  //const infowindowContent = $('#infowindow-content');
+  const politicalBoundary = 'political'; // Bee.place just contains Administrative/Political location
   const geoCoding = function() {
-    geometry=false;
     var autocomplete = new google.maps.places.Autocomplete(input);
```

Comment:
let's use `let` rather than `var`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
Done! Good contribution I didn't know it